knowledge/animals/tabby_cat/qna.yaml

```diff
@@ -0,0 +1,67 @@
+created_by: sborenst,jregehr
+version: 2
+domain: tabby cat
+seed_examples:
+ - answer: domestic cat
+   question: What kind of cat is a tabby?
+ - answer: |
+      A tabby cat has a distinctive M-shaped marking on their forehead, stripes by their
+      eyes and across their cheeks, along their back, around their legs
+      and tail, and characteristic striped, dotted, lined, flecked,
+      banded, or swirled patterns on the body: neck, shoulders, sides,
+      flanks, chest, and abdomen. The four known distinct patterns,
+      each having a sound genetic explanation, are the mackerel, classic
+      or blotched, ticked, and spotted tabby patterns.
+   question: How can you tell if a cat is a tabby cat?
+ - answer: |
+      The English term tabby originally referred to "striped silk taffeta",
+      from the French word tabis, meaning "a rich watered silk".
+   question: What is the origin of the name tabby?
+ - answer: |
+      The four known distinct patterns, each having a sound genetic
+      explanation, are the mackerel, classic, ticked, and spotted tabby patterns
+   question: What are the patterns that a tabby cat may have?
+ - answer: Personality and aggression vary widely from cat to cat, and is multifactorial.
+   question: Do tabby cats have a specific personalities?
```

Comment:
> question: Do tabby cats have a specific personalities?
```   question: Do tabby cats have specific personalities?
```

Reply:
I made the requested change and updated the PR